packages/components/color-picker/src/color-picker.vue

```diff
@@ -217,6 +217,11 @@ function setShowPicker(value: boolean) {
 
 const debounceSetShowPicker = debounce(setShowPicker, 100)
 
+function show() {
+  if (colorDisabled.value) return
+  debounceSetShowPicker(true)
```

Comment:
Manual mode may not require debounce processing.

Reply:
Use `setShowPicker`.